src/astro/files/operators/files.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+
+from astro.files.locations import create_file_location
+
+
+class ListFileOperator(BaseOperator):
+    """List the file available at path and storage
+
+    :param task_id: The task id for uniquely identify a task in a DAG
+    :param path: A path pattern for which you want to get a list of file
```

Comment:
It should probably contain examples of what are the things that you can pass e.g bucket like: `s3://bucket/object` or even a pattern like `gs://bucket/object*.csv`